### DISCUSSION TITLE
What is the recomended size for cover images (Clean Blog theme)?

### DISCUSSION THREAD

**Discussion by TechWatching** [statiqdev/Discussions#199] [Q&A] [answered]
I am wondering what is the recommended size I should use for my cover images.

**Comment by daveaglick** [comment, marked as answer]
Good question. The cover image will scale to the browser width, so the tradeoff is mostly about resolution vs. file size. Too small an image and it'll get scaled up on larger screens, potentially looking bad in the process. To large an image and it'll look great in all screen sizes, but will potentially be large, use lots of bandwidth, and be slow to load.

Statcounter has some stats on this: https://gs.statcounter.com/screen-resolution-stats

From that, it looks like 1920px is the most popular width, and most other popular widths are lower, so if you want the best-looking cover image you probably want to target that. If you want to cut down on size a bit, 1440px is probably another safe width.

The height of the cover image is variable depending on the responsible screen resolution, but making it half or even a third of the width is probably safe. It doesn't have to be perfect since the image will get stretched (or shrunk) in whatever dimensions it needs to in order to keep the same aspect ratio.

**Reply by TechWatching** [reply]
Great answer, thanks.
